enterprise-suite/values.yaml

```diff
@@ -59,7 +59,7 @@ prometheusMemoryRequest: 250Mi
 # Persistent Volumes
 #
 # Set to true to use emptyDir for all volumes, disabling persistent volumes.
-useEmptyDirVolumes: false
+useEmptyDirVolumes: true
 # Default StorageClass to use for persistent volumes
```

Comment:
Does the StorageClass comment need something along the lines of "ignored if `useEmptyDirVolumes` is `true`"?